release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -2182,3 +2182,109 @@ xref:ocp-311-technology-preview[Technology Preview] and now generally available.
+
+* Health Check playbooks would fail at checking `Elasticsearch` because the exec call would not specify a container. The call failed because the output included incorrectly formatted JSON text. Now, the target container is included in the `exec` call and the Health Check succeeds.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1660956[*BZ#1660956*])
+
+* An error in `glusterfs` pod mount points prevented the use of `gluster-block`. As a result, the provisioner would fail to create devices. The mount points have now been updated and the provisioning process succeeds as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1662312[*BZ#1662312*])
+
+* The `openshift-ansible` package was incorrectly checking if a value in the `etcd-servers-overrides` was a valid path. Some values were considered invalid by the `openshift-ansible-3.11.51-2.git.0.51c90a3.el7.noarch` package. Now, `etcd-servers-overrides` does not contain paths, and is ignored during path checks.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1666491[*BZ#1666491*])
+
+* `etcd` non-master host nodes were excluded from upgrades. Now, `etcd` host nodes are able to be upgraded.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1668317[*BZ#1668317*])
+
+* The Ansible variable `openshift_master_image_policy_allowed_registries_for_import` was incorrectly parsed, causing a corrupted `master-config.yaml` file. Now, the `openshift_master_image_policy_allowed_registries_for_import`
+variable is correctly parsed and a simple registry image policy can be set as expected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1670473[*BZ#1670473*])
+
+* The playbooks and manual configuration steps to redeploy router certificates were replaced with service serving certificates secret. This would overwrite or miss the router wild certificates secret, causing certificate errors due to incorrect certificates redeployed. Now, the playbooks and manual redeployment steps do not overwrite router certificates secret. The router certificates are redeployed based on the specified sub domain or customer certificates.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1672011[*BZ#1672011*])
+
+* The `ImageStream` used in the `BuildConfig` editor did not have edit properties, causing runtime errors in the `BuildConfig` editor. Now, the editor is initializing tags and objects, even if `ImageStream` in the `BuildConfig` is missing or if the user does not have the correct permissions to use it.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1672904[*BZ#1672904*])
+
+* Master pods did not match time zones with worker nodes, which led to errors in logging timestamps. Now the host's timezone configuration is mounted into the control plane pods.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1674170[*BZ#1674170*])
+
+* When a cluster was installed, the username in the loopback kubeconfig is the same as the hostname of the master. Now, the variable in the playbook is changed to a different value.
```

Comment:
s/username/user name
s/hostname/host name  
(per IBM style guide)

Reply:
Ah, that's news to me. I was following the Linux style syntax. Thanks for that heads up!